va/compat_win32.h

```diff
@@ -46,6 +46,15 @@ typedef LONG NTSTATUS;
 
 typedef unsigned int __uid_t;
 
+#if _MSC_VER
+#define getenv _getenv
```

Comment:
So, Windows does define getenv in `<stdlib.h>`: https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/getenv-wgetenv?view=msvc-170. And since code did built without error we were fetching this definition.

It seems that's important in which order we include header files. If compat_win32.h will by any chance be included before stdlib.h, then your trick to replace getenv won't work. Maybe we need include stdlib.h here in compat_win32.h explicitly to make sure we always replace getenv definition?

Reply:
You're right. It is picking the include stdlib.h from va.c before compat_win32. For the Windows build path, I moved the stdlib.h inclusion out from va.c and into compat_win32.c instead as you suggested to replace the definition.